src/bin/vip-import-sql.js

```diff
@@ -214,12 +214,13 @@ command( {
 		console.log( `         to: ${ chalk.cyan( domain ) }` );
 		console.log( `       site: ${ app.name } (${ formatEnvironment( opts.env.type ) })` );
 		if ( searchReplace && searchReplace.length ) {
-			const searchAndReplaceParams = searchReplace.split( ',' );
-			console.log(
-				`        s-r: ${ chalk.blue( searchAndReplaceParams[ 0 ] ) } -> ${ chalk.blue(
-					searchAndReplaceParams[ 1 ]
-				) }`
-			);
+			const searchReplaceList =
```

Comment:
It would be nice to consolidate this logic in the command portion below and just pass any derived strings we want to print to the args handler here.

Reply:
That said, I'm curious if we really need both.

Reply:
Refactored! I left both in since the initial summary is a bit hard to see once the progress logs get long!